packages/fuels-contract/src/script.rs

```diff
@@ -409,8 +410,9 @@ impl Script {
         if receipts
             .iter()
             .any(|r|
-                matches!(r, Receipt::ScriptResult { result, .. } if *result != ScriptExecutionResult::Success)
-        ) {
+                matches!(r, Receipt::ScriptResult { result, .. } if *result != ScriptExecutionResult::Success) |
+                    matches!(r, Receipt::Panic { reason, .. } if *reason.reason() == PanicReason::ContractNotInInputs )
```

Comment:
I'm pretty sure the execution result should != `ScriptExecutionResult::Success` if there is missing contracts. No need for an additional check.